query-engine/core/src/interpreter/query_interpreters/nested_read.rs

```diff
@@ -14,6 +14,8 @@ pub async fn m2m<'a, 'b>(
     let parent_field = &query.parent_field;
     let child_link_id = parent_field.related_field().linking_fields();
 
+    println!("m2m: {:?}", parent_result);
```

Comment:
Remove all printlns

Reply:
Of course, but I am not done with this PR yet.